config/crd/bases/crd.argoproj.io_rollouts.yaml

```diff
@@ -0,0 +1,3630 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
I wasn't sure how to add this CRD via kustomize. Wondering if there was a way to autogenerate this into bases folder?

Reply:
I tried to write test to create a Rollout object and then create a PodAccessTemplate pointing to the Rollout and PodAccessRequest. 

```
 failed to get API group resources: unable to retrieve the complete list of server APIs: argoproj.io/v1alpha1: the server could not find the requested resource
```

I read through the docs for this and tried to apply this but it seems that the cluster itself does not know what a Rollout object is when the client attempts to create it.

A workaround was to manually add the crd to the crd base path. 
https://github.com/diranged/oz/blob/main/internal/builders/podaccessbuilder/suite_test.go#L62.
I wasnt sure if theres other options ?